packages/kbn-monaco/src/console/lexer_rules/shared.ts

```diff
@@ -14,11 +14,13 @@ export const consoleSharedLanguageConfiguration: monaco.languages.LanguageConfig
   brackets: [
     ['{', '}'],
     ['[', ']'],
+    ['"""', '"""\n'],
   ],
   autoClosingPairs: [
     { open: '{', close: '}' },
     { open: '[', close: ']' },
     { open: '"', close: '"' },
+    { open: '"""', close: '"""' },
```

Comment:
This makes the editor automatically add a closing `"""` when the user types in an opening `"""`.